krummstab/config.py

```diff
@@ -16,9 +13,14 @@ def __init__(self, config_paths: list[Path]) -> None:
         data = {}
         for path in config_paths:
             logging.info(f"Reading config file '{path}'")
-            data.update(json.loads(path.read_text(encoding="utf-8")))
-        config_schema = json.loads(resources.files(schemas).joinpath("config-schema.json").read_text(encoding="utf-8"))
-        jsonschema.validate(data, config_schema, jsonschema.Draft7Validator)
+            try:
+                data.update(utils.read_json(path))
+            except FileNotFoundError:
+                logging.warning(f"File '{path}' is missing.")
+        config_schema = utils.read_json(
+            resources.files(schemas).joinpath("config-schema.json")
```

Comment:
Use `read_text` from libimport.resources to get string which is then passed to utils.read_json.